synthesizer/src/store/transaction/execution.rs

```diff
@@ -37,7 +37,7 @@ pub trait ExecutionStorage<N: Network>: Clone + Send + Sync {
     type IDMap: for<'a> Map<'a, N::TransactionID, (Vec<N::TransitionID>, bool)>;
     /// The mapping of `transition ID` to `transaction ID`.
     type ReverseIDMap: for<'a> Map<'a, N::TransitionID, N::TransactionID>;
-    /// The mapping of `transaction ID` to `(global state root, (optional) inclusion proof)`.
+    /// The mapping of `transaction ID` to `(global state root, (optional) proof)`.
```

Comment:
Just in case you overlooked it, since there is not just an inclusion proof might want to rename: https://github.com/AleoHQ/snarkVM/pull/1510/files#diff-15129d118f13be5484e52c96b29f3e95e65f712150fed731d8ea8e02d1d31db5R43

Reply:
Yeah I debated making this change here, mainly because its quite a large changeset already.

Will address in a future PR.